R/between.R

```diff
@@ -22,18 +22,18 @@ inrange <- function(x,lower,upper,incbounds=TRUE) {
   subject = setDT(list(l=lower, u=upper))
   ops = if (incbounds) c(4L, 2L) else c(5L, 3L) # >=,<= and >,<
   verbose = getOption("datatable.verbose")
-  if (verbose) {last.started.at=proc.time()[3];cat("forderv(query) took ... ");flush.console()}
```

Comment:
@mattdowle what about two functions, `timestart` and `timetaken`? `timestart` would accept a message as input, `cat` it, `flush.console()`s, and return the `proc.time()`; `timetaken` takes the result of `timestart`, `cat`s it, and `flush.console()`s, and returns nothing.

Reply:
Yes separate PR.  Sometimes there are several nested timing blocks in play.  Storing the `started.at` time in a variable and then passing that (or the appropriate one) to `timetaken` I think I like the balance of using a common function along with the flexibility of using it in different ways depending on the circumstances. i.e. current `timetaken()` approach, but actually using it here. IIUC.